src/main/java/io/kokuwa/maven/helm/UploadMojo.java

```diff
@@ -167,6 +203,39 @@ private void uploadSingle(Path chart) throws MojoExecutionException, IOException
 		connection.disconnect();
 	}
 
+	private boolean verifyUpload(Path chartPath) throws MojoExecutionException {
+		ObjectMapper MAPPER = new YAMLMapper()
+				.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
+		String chartName;
+		try {
+			chartName = MAPPER.readValue(chartPath.toFile(), HelmChart.class).getName();
+		} catch (IOException e) {
+			throw new MojoExecutionException("Unable to read chart from " + chartPath, e);
+		}
```

Comment:
gets the Chartpath and uses YamlMapper to pull the chart name field (which will be used to run helm show chart..."